Framework/Algorithms/src/CalculateTransmissionBeamSpreader.cpp

```diff
@@ -140,11 +139,9 @@ void CalculateTransmissionBeamSpreader::exec() {
 
   // Note: Replaced PARALLEL_SECTION with this OMP for loop, due to occasional
   // unexplained segfault.
-  std::vector<MatrixWorkspace_sptr> in_ws, out_ws(4);
-  in_ws.push_back(sample_scatterWS);
-  in_ws.push_back(direct_scatterWS);
-  in_ws.push_back(sample_spreaderWS);
-  in_ws.push_back(direct_spreaderWS);
+  std::vector<MatrixWorkspace_sptr> in_ws{sample_scatterWS, direct_scatterWS,
+                                          sample_spreaderWS, direct_spreaderWS},
+      out_ws(4);
```

Comment:
I know this is an unrelated change, but could you please declare `out_ws` separately?